Workflow.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Mar 23 12:30:17 2016
+
+@author: mina
+"""
+
+import yaml
+
+class Workflow:
```

Comment:
I think rather than `Workflow` we could call the entire object from #4 a `WorkflowConfig` which inherits from `object`.

```
class WorkflowConfig(object):
...
```